website/app/validators.php

```diff
@@ -24,7 +24,10 @@
 }, _('This language is not supported'));
 
 $validator->addValidator('ciphered_password', function ($field, $input, $param = null) {
-    return substr($input[$field], 0, 7) === '$2a$11$';
+    $expectedPasswordPrefix = '$2a$'.sprintf("%02d", GK_PASSWORD_HASH_ROTATION).'$';
+    $expectedMinimalHashRotation = GK_PASSWORD_HASH_ROTATION >= 8;
```

Comment:
May this `8` be a configurable variable `GK_PASSWORD_MIN_HASH_ROTATION`?

Reply:
I've choosed min=10, max=99